hoodie-common/src/main/java/com/uber/hoodie/common/BloomFilter.java

```diff
@@ -37,33 +38,49 @@
    */
   public static final double LOG2_SQUARED = Math.log(2) * Math.log(2);
 
-  private org.apache.hadoop.util.bloom.BloomFilter filter = null;
+  private org.apache.hadoop.util.bloom.BloomFilter bloomFilter = null;
 
-  public BloomFilter(int numEntries, double errorRate) {
-    this(numEntries, errorRate, Hash.MURMUR_HASH);
+  private org.apache.hadoop.util.bloom.DynamicBloomFilter dynamicBloomFilter = null;
```

Comment:
would be curious to see which other projects use this.. if its easy to do that..

Reply:
Hbase -> https://hbase.apache.org/devapidocs/org/apache/hadoop/hbase/util/BloomFilter.html
Looked through Hbase code and they have implemented their own BloomFilters based on the algorithms in the above class.
Cassandra -> https://docs.datastax.com/en/cassandra/3.0/cassandra/operations/opsTuningBloomFilters.html
They don't support dynamic bloom filters, instead force rewriting the bloom filter.

Reply:
Accumulo seems to be implementing something.. 
https://github.com/apache/accumulo/blob/master/core/src/main/java/org/apache/accumulo/core/bloomfilter/DynamicBloomFilter.java as well.

Reply:
https://github.com/apache/accumulo/blob/master/core/src/main/java/org/apache/accumulo/core/bloomfilter/DynamicBloomFilter.java also has one.. It'd be good to know tradeoffs each made. esp HBase and Accumulo